src/interpreter.rs

```diff
@@ -169,8 +169,8 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
 
                 for stmt in &block_data.statements {
                     self.log(0, || print!("{:?}", stmt));
-                    let mir::StatementKind::Assign(ref lvalue, ref rvalue) = stmt.kind;
-                    let result = self.eval_assignment(lvalue, rvalue);
+                    let mir::StatementKind::Assign(ref l_value, ref r_value) = stmt.kind;
+                    let result = self.eval_assignment(l_value, r_value);
```

Comment:
@llogiq Considering these names are only used on two *adjacent* lines, I'd rather stick with the existing names to be consistent with the rest of the code.